apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/interceptor/enhance/ClassEnhancePluginDefine.java

```diff
@@ -130,14 +134,13 @@
                 if (isBootstrapInstrumentation()) {
                     newClassBuilder = newClassBuilder.constructor(constructorInterceptPoint.getConstructorMatcher())
                                                      .intercept(SuperMethodCall.INSTANCE.andThen(MethodDelegation.withDefaultConfiguration()
-                                                                                                                 .to(BootstrapInstrumentBoost
-                                                                                                                     .forInternalDelegateClass(constructorInterceptPoint
-                                                                                                                         .getConstructorInterceptor()))));
+                                                                                                                 .to(new BootstrapConstructorInter(
+                                                                                                                     constructorInterceptPoint.getConstructorInterceptor()))));
```

Comment:
ByteBuddy doesn't recommend the instance-delegation explicitly for the bootstrap process. Its author recommends the static level delegate. So I changed to the delegator generation solution.
I wouldn't have the same experiences as he does.

Reply:
I don't have a full understanding of bootstrap level instrumentation. I would like to follow what he recommended.